webapp/pages/index.vue

```diff
@@ -73,6 +74,14 @@ export default {
     },
   },
   methods: {
+    bottomPageHandler($state) {
+      if (this.$apollo.loading) return
+      this.showMoreContributions(() => {
+        setTimeout(() => {
+          $state.loaded()
+        }, 1000)
+      })
+    },
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-03T16:23:47Z" title="Wednesday, July 3rd 2019, 6:23:47 pm +02:00">Jul 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -86,6 +99,7 @@ export default {
         },
         // Transform the previous result with new data
         updateQuery: (previousResult, { fetchMoreResult }) => {
+          this.hasMore = fetchMoreResult.Post && fetchMoreResult.Post.length > 0
```


```suggestion
          this.hasMore = fetchMoreResult.Post && fetchMoreResult.Post.length >= this.pageSize
```
Would give better user experience.